pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -155,82 +155,122 @@ class SierraMergeCandidatesTest
         )
       )
 
-      transformer.getMergeCandidates(bibData) shouldBe singleMiroMergeCandidate(
-        miroID)
+      transformer.getMergeCandidates(bibData) shouldBe
+        singleMiroMergeCandidate(miroID)
+    }
+
+    // - - - - - - -  089 fields - - - - - - -
+    it("merges a MIRO ID for a picture with MARC tag 089 subfield a if there is no 962 tag") {
+      val bibData = createMiroPictureWith(
+        varFields = create089subfieldsWith(List("V 13889"))
+      )
+
+      transformer.getMergeCandidates(bibData) shouldBe
+        singleMiroMergeCandidate(
+          miroID = "V0013889",
+          reason = "Single page Miro/Sierra work (secondary source)")
+    }
+
+    it("merges a MIRO ID for a digital image with MARC tag 089 subfield a if there is no 962 tag") {
```

Comment:
These two tests are the same, we haven't added the work type to match the test description